docs/02-tutorial.mdx

```diff
@@ -0,0 +1,281 @@
+
+User interfaces for the blindnet devkit (aka _[Privacy Components for the Web](https://github.com/blindnet-io/privacy-components-web)_) are delivered as standard Web Components available on both npm and a CDN.
+
+As a consequence, installing the blindnet devkit web interfaces doesn't involve any specificity, as you can install and use them as you would with any other standard Web Component.
+
+:::info
+
+See [Using Web Components](/docs/interfaces/using-web-components) for more information on how to integrate Web Components in your existing project.
+
+:::
+
+In the interest of simplicity and clarity, let's see how to integrate these components into a simple static HTML page, starting with the Privacy Request Capture Interface (PRCI).
+
+PRCI is a simple interface allowing your users to assert their privacy rights by sending you a simple yet standardized [Privacy Request](/docs/references/lexicon#privacy-request). It is an entry point for your user to privacy, not only allowing you to cover GDPR and other privacy-related legislations but also and more importantly, to build trust by showing your users their privacy rights are tangibly handled, enforced and respected.
+
+To integrate this interface into your web page, you'll first need to load the associated bundle (included in the `@blindnet/prci` npm package) from the unpkg CDN by adding a `<script>` tag to your HTML document:
+
+```html
+<script src="https://unpkg.com/@blindnet/prci/dist/index.core.min.js?module" type="module"></script>
+```
+
+:::info
+
+Here, we use the `core` module build for performance improvement.
+
+As this build only integrates the _core_ parts of the library, external dependencies like [Lit](https://lit.dev) will only be loaded once, even after we add other privacy components to this same web page.
+
+Your specific context might be different. This is why each Privacy Component for the Web includes different builds and bundles covering any specific needs, as detailed in the [User Interfaces - Install](/docs/interfaces/install) section.
+
+:::
+
+Then, you only need to add the associated entry-point custom element `<bln-priv-request>` to your HTML document:
+
+```html
+<h2 id="priv-request">Make a privacy request</h2>
+
+<bldn-priv-request></bldn-priv-request>
+```
+
+Leading us to the following HTML file:
+
+```html
+<!DOCTYPE html>
+<html lang="en-GB">
+  <head>
+    <meta charset="utf-8" />
+    <title>Privacy Components for the Web - Minimal Static Demo</title>
+  </head>
+  <body>
+    <h1>Privacy Components for the Web - Minimal Static Demo</h1>
+
+    <h2>Assert Your Privacy Rights</h2>
+
+    <bldn-priv-request></bldn-priv-request>
+
+    <script src="https://unpkg.com/@blindnet/prci/dist/index.core.min.js?module" type="module" ></script>
+  </body>
+</html>
+```
+
+:::info Demo
+
+This simple static demo can be tested live on [blindnet-io.github.io](https://blindnet-io.github.io/privacy-components-web/demos/devkit-simple-tutorial/demos/static/).
```

Comment:
Page not found for this url

Reply:
That's indeed an error of inattention.

```suggestion
This simple static demo can be tested live on [blindnet-io.github.io](https://blindnet-io.github.io/privacy-components-web/demos/static/).
```

BTW, this demo is broken on `main` currently, but it has been fixed in the next release (see https://github.com/blindnet-io/privacy-components-web/pull/112).